core/web/log_controller.go

```diff
@@ -87,15 +87,13 @@ func (cc *LogController) Patch(c *gin.Context) {
 	svcs = append(svcs, "Global")
 	lvls = append(lvls, cc.App.GetConfig().LogLevel().String())
 
-	// NOTE: This doesn't actually do anything right now since sqlx doesn't support query logging
-	// TODO: Consider removing it?
-	// See: https://app.shortcut.com/chainlinklabs/story/21518/log-sql-env-var-now-does-nothing
 	if request.SqlEnabled != nil {
 		if err := cc.App.GetConfig().SetLogSQLStatements(*request.SqlEnabled); err != nil {
 			jsonAPIError(c, http.StatusInternalServerError, err)
 			return
 		}
 	}
+
```

Comment:
The previous update needs to propagate the config change downstream to anything already instantiated as well. See how `App.SetLogLevel` updates the config as well as the live root logger (which is linked internally to all descendant loggers).

Reply:
Fixed, made is similar to `App.SetLogLevel`, please check.